### ISSUE TITLE
Log display, training loss ? validation loss ?

### ISSUE THREAD

**Issue by CamilleMaurice** [open-mmlab/mmskeleton#307]
Hi everyone,
I'm not exactly sure about how I should interpret the log during training.

For example, here I assume the loss is the training loss : 
_INFO:mmcv.runner.runner:Epoch [17][100/945]	lr: 0.00500, eta: 0:37:08, time: 0.060, data_time: 0.006, memory: 859, **loss**: 0.0006
INFO:mmcv.runner.runner:Epoch [17][200/945]	lr: 0.00500, eta: 0:37:03, time: 0.056, data_time: 0.002, memory: 859, **loss**: 0.0010
INFO:mmcv.runner.runner:Epoch [17][300/945]	lr: 0.00500, eta: 0:36:58, time: 0.056, data_time: 0.002, memory: 859, **loss**: 0.0014
INFO:mmcv.runner.runner:Epoch [17][400/945]	lr: 0.00500, eta: 0:36:53, time: 0.056, data_time: 0.002, memory: 859, **loss**: 0.0007_
I

But after, I'm not sure whether this is the traninng loss or the validation loss. I suppose it is the validation loss (much higher). And the top1 accuracy is on the training set or the validation set ? 
As the loss is much higher I suppose it is the validation loss and the top1 is the top1 accuracy on the validation set, but I'm not sure.

_INFO:mmcv.runner.runner:Epoch(train) [20][24]	**loss**: 1.0869, **top1**: 0.6927, top2: 0.8229, top5: 0.9635_

Is it possible to display both training loss/validation loss during training ? I looked in /mmskeleton/processor/recognition.py but I'm not sure how to do.

Thank you

**Comment by CamilleMaurice** [comment]
I think I found the answer. To state clearly the issue often helps to solve it right ?
I guess my assumptions were correct since I managed to display the boolean 'train_mode' inside the batch_processor function.

_def batch_processor(model, datas, train_mode, loss): 
  ....
#    if not train_mode:
 log_vars['top1'] = topk_accuracy(output, label)
log_vars['train_mode'] = train_mode_
